portfolio.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="portfolio.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div class="first">
+      <header class="bg-white">
+        <div class="logo">
+          <a href="#">My Logo</a>
+        </div>
+        <nav>
+          <div class="menu">
+            <img src="images/appbar.png" alt="menu button" />
+          </div>
+        </nav>
+      </header>
+        <section class="headline bg-white">
+          <h1 class="title">I’m Jery Glad to see you!</h1>
+          <p class="details">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET’S CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="#"><img src="images/social-media-icon/tw.png" alt="Twitter"/></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/social-media-icon/linkedin.png"alt="LinkedIn"/></a>
+              </li>
+              <li><a href="#"><img src="images/social-media-icon/medium.png" alt="Medium" /></a>
+              </li>
+              <li><a href="#"><img src="images/social-media-icon/github.png" alt="Github"/></a>
+              </li>
+              <li><a href="#"><img src="images/social-media-icon/v.png" alt="V"/></a>
+              </li>
+            </ul>
+          </div>
+        </section>
+      
```

Comment:
- [ ] Nice work 👍🏿  so far on your portfolio website but noticed that there are some missing closing tags. It's important to include these tags to ensure the proper structure and functionality of your webpage. Here are the closing tags that you need to add:

 - [ ] Closing tag for the <div class="first"> element: </div>
- [ ] Closing tag for the <body> element: </body>
- [ ] Closing tag for the <html> element: </html>
By adding these closing tags, your HTML code will be well-formed and valid. This will help prevent any rendering issues and ensure that your website displays correctly across different browsers.

Reply:
@Cabraham1 Ok am done with it, thank you